actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -379,35 +383,43 @@ private String getCreatedBy(Request request) {
     List<String> mappedColumns = new ArrayList<>();
     csvColumns.forEach(
         column -> {
-          if (column.equalsIgnoreCase("email id")) {
-            mappedColumns.add(JsonKey.EMAIL);
-          }
-          if (column.equalsIgnoreCase("phone number")) {
-            mappedColumns.add(JsonKey.PHONE);
-          }
-          if (column.equalsIgnoreCase("state provided ext. id")) {
-            mappedColumns.add(JsonKey.USER_EXTERNAL_ID);
-          }
-          if (column.equalsIgnoreCase("")) {
-            mappedColumns.add(JsonKey.ORG_EXTERNAL_ID);
-          }
-          if (column.equalsIgnoreCase("status")) {
-            mappedColumns.add(JsonKey.INPUT_STATUS);
-          }
-          if (column.equalsIgnoreCase("diksha uuid")) {
-            mappedColumns.add(JsonKey.USER_ID);
-          }
-          if (column.equalsIgnoreCase("channel")) {
-            mappedColumns.add(column);
-          }
-          if (column.equalsIgnoreCase("school name")) {
-            mappedColumns.add("schoolName");
-          }
-          if (column.equalsIgnoreCase("school udise id")) {
-            mappedColumns.add("schoolUdiseId");
-          }
-          if (column.equalsIgnoreCase("diksha sub-org id")) {
-            mappedColumns.add("subOrgId");
+          switch (column) {
```

Comment:
We discussed these mapping of fields and columns must be externalized. Pls use external.properties file like mechanism. @reshmi-nair will come up with a way to store these as part of tenant preferences.